c/CMakeLists.txt

```diff
@@ -50,9 +50,11 @@ cmake_minimum_required(VERSION 2.8)
 set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 
 # Some compiler options used globally
-if (NOT MSVC)
+if (MSVC)
+  set(CMAKE_C_FLAGS "/Wall /WX ${CMAKE_C_FLAGS}")
+else (MSVC)
```

Comment:
I personally prefer else () since there is no test going on here, and this is actually the NOT MSVC case. I find it visually misleading when I am skimming a file.